### DISCUSSION TITLE
Relationship data

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6250] [Help]
Hello everyone.
I have two tables (Equipments and Pieces). Each equipment (row) in the Equipments table has several pieces (rows) in the Pieces table.
I want when I click on an equipment, to display in a new page just the pieces related to the clickable equipment

**Comment by melohagan** [comment]
Hello @rafikApp 

I believe this will help: https://docs.budibase.com/docs/fetching-data#displaying-relational-data

**Reply by rafikApp** [reply]
I have already read this article, but it didn't help me.
